counterpartylib/lib/arc4.py

```diff
@@ -0,0 +1,8 @@
+import binascii
+from Crypto.Cipher import ARC4
+
+def init_arc4(seed):
```

Comment:
That's odd I don't remember adding or needing this. I must have picked it up with some merge against another branch I needed. My guess is that branch as not made it into dev yet. @rubensayshi do you know more about this code?